app/models/theme_modifier_set.rb

```diff
@@ -50,6 +50,26 @@ def self.resolve_modifier_for_themes(theme_ids, modifier_name)
     end
   end
 
+  def topic_thumbnail_sizes
+    array = super
```

Comment:
use of super is a bit surprising here... `read_attribute(:topic_thumbnail_sizes)` is a bit less surprising